modules/core/src/main/java/org/apache/synapse/config/xml/CallMediatorFactory.java

```diff
@@ -76,6 +77,9 @@
     private static final QName CALL_Q = new QName(XMLConfigConstants.SYNAPSE_NAMESPACE, "call");
     private static final QName ENDPOINT_Q = new QName(XMLConfigConstants.SYNAPSE_NAMESPACE, "endpoint");
     private static final QName BLOCKING_Q = new QName("blocking");
+    private static final QName ATT_INIT_AXI2_CLIENT_OPTIONS = new QName("initAxis2ClientOptions");
+    private static final QName ATT_AXIS2XML = new QName("axis2xml");
+    private static final QName ATT_REPOSITORY = new QName("repository");
```

Comment:
init client options improvement is not related to group endpoint support feature, so please spilt into two PRs